flink/src/test/java/org/apache/iceberg/flink/data/RandomData.java

```diff
@@ -88,6 +93,34 @@ public Row next() {
     };
   }
 
+  private static Iterable<RowData> generateRowData(Schema schema, int numRecords,
```

Comment:
We will remove this `RandomData` class in the future (Once RowData parquet reader & writer is ready),   we've already had the `RandomRowData` to generate `Iterable<RowData>`, you may want to rebase master branch now.  (so we don't have to define a `generateRowData` here now).
It will generate `Record` firstly, then convert them into `RowData`, in this way we could assert the RowData with Record,  it can be more rigorous to determine the correctness of the data.

Reply:
Sorry, this is legacy useless codes.